src/views/Player.vue

```diff
@@ -315,8 +324,15 @@ export default {
       // update response on S3
       this.uploadJson()
 
-      this.player.currentTime = (currQuesIndex == 0) ? 0 : this.times[currQuesIndex - 1];
+      // after revise is clicked, make the user land just next to the marker
+      // and not on the marker so that question pops up again
+      this.player.currentTime = 
+        (currQuesIndex == 0) ? 0 : (this.times[currQuesIndex - 1] + (popUpPrecisionTime)/1000);
+
       this.player.play();
+
+      // logging for testing
+      console.log("Answer revised");
```

Comment:
Revised the `revise` button functionality. When clicked revise, the user should not land at some previous timestamp where he will see the previous question pop-up again. We want to be just a little ahead.